VirtoCommerce.Platform.Web/Modularity/ManifestModuleCatalog.cs

```diff
@@ -170,25 +170,50 @@ public override void Validate()
             return result;
         }
 
+        private static void CopyLocalizationAssemblies(string sourceDirectoryPath, string targetDirectoryPath)
+        {
+            var languageCodes = ConfigurationHelper.GetAppSettingsValue("VirtoCommerce:Localization:ServerLocalizationCodes");
```

Comment:
Need to provide default value "string.Empty" here, to avoid NullReference.
Maybe there is a method in `ConfigurationHelper` that gets value or default.

Reply:
Also please name it `cultureNames`

Reply:
done